packages/modal/src/components/Modal.module.css

```diff
@@ -0,0 +1,110 @@
+@keyframes fadeIn {
+  0% {
+    background-color: var(--swui-hidden);
+  }
+  100% {
+    background-color: var(--swui-modal-overlay-bg-color);
+  }
+}
+
+@keyframes appear {
+  0% {
+    top: -8px;
+    opacity: 0;
+  }
+  100% {
+    top: 8px;
+    opacity: 1;
+  }
+}
+
+.Modal {
+  max-width: 100%;
+  outline: none;
+  pointer-events: none;
+}
+
+.Overlay {
+  --swui-modal-animation-time: var(--swui-animation-time-fast);
+  --swui-modal-overlay-bg-color: rgba(0, 0, 0, 0.5);
+  --swui-modal-padding: 8px;
+
+  position: fixed;
+  z-index: 10;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: var(--swui-modal-overlay-bg-color);
+  overflow: auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  padding: var(--swui-modal-padding);
+
+  animation: fadeIn var(--swui-modal-animation-time)
+    cubic-bezier(0.645, 0.045, 0.355, 1) both;
+
+  @media print {
+    background-color: rgba(255, 255, 255, 1);
+  }
+}
+
+.Content {
+  --swui-modal-animation-time: var(--swui-animation-time-fast);
+  --swui-modal-content-bg-color: var(--swui-white);
+  --swui-modal-width: 960px;
+  --swui-modal-header-border-color: #e4e5e9;
+
+  position: relative;
+  display: block;
+
+  background: var(--swui-modal-content-bg-color);
+  box-shadow: var(--swui-shadow-modal);
+  pointer-events: all;
+
+  width: var(--swui-modal-width);
+  max-width: 100%;
+
+  animation: appear var(--swui-animation-time-fast)
+    cubic-bezier(0.645, 0.045, 0.355, 1) both;
+
+  @media print {
+    box-shadow: none;
+    top: 0;
+  }
+
+  :focus {
+    outline: 0;
+  }
+
+  &.isDraggable {
+    :global(.draggable-modal-handle) {
+      cursor: move;
+    }
+  }
+
+  .Header {
+    padding: 16px 16px 16px 24px;
+    display: flex;
+    justify-content: space-between;
+    font-size: 16px;
+    align-items: center;
+    border-bottom: 1px solid var(--swui-modal-header-border-color);
+  }
+
+  .HeaderCloseButton {
```

Comment:
We have used `headerCloseButton` style names for all other CSS.
I kind of like this though, but I think we should use same thing everywhere.
@lindskogen what do you think?

Reply:
Also, don't do custom buttons :D

Reply:
Maybe you can use FlatButton now, since the new buttons have been merged?